tensorflow/core/kernels/dml_cwise_ops.cc

```diff
@@ -401,336 +285,248 @@ class DmlUnaryScaleBiasKernel : public DmlKernel {
+REGISTER_DML_COMPOSITE_UNARY_FLOAT_KERNEL(IsFinite, !(dml::IsNaN(x) ||
+                                                      dml::IsInfinity(x)))
 // TODO(b/25387198): A special kernel exists for int32 (see
 // cwise_op_not_equal_to_1.cc).
-REGISTER_DML_COMPOSITE_BINARY_ALL_TYPES_EXCEPT_INT32_KERNEL(NotEqual, x != y,
-                                                            kNchwDimensionCount)
+REGISTER_DML_COMPOSITE_BINARY_ALL_TYPES_EXCEPT_INT32_KERNEL(NotEqual, x != y, 8)
 // TODO(b/25387198): A special kernel exists for int32 (see
 // cwise_op_less_equal.cc).
 REGISTER_DML_COMPOSITE_BINARY_ALL_TYPES_EXCEPT_INT32_KERNEL(LessEqual, x <= y,
-                                                            kNchwDimensionCount)
+                                                            8)
 // TODO(b/25387198): A special kernel exists for int32 (see
 // cwise_op_greater_equal.cc).
 REGISTER_DML_COMPOSITE_BINARY_ALL_TYPES_EXCEPT_INT32_KERNEL(GreaterEqual,
-                                                            x >= y,
-                                                            kNchwDimensionCount)
+                                                            x >= y, 8)
 // TODO: Register this operator for all types (except int32) when FloorDiv is
 // added to DML. dml::Floor(x / y) works for float types, but it doesn't work
 // for integer types since DML truncates towards zero.
 // TFDML #25977645
 // TODO(b/25387198): A special kernel exists for int32 (see
 // cwise_op_floor_div.cc).
-REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(FloorDiv, dml::Floor(x / y),
-                                           kNchwDimensionCount)
-REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(SigmoidGrad, y* x*(1 - x),
-                                           kNchwDimensionCount)
-REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(TanhGrad, y*(1 - x * x),
-                                           kNchwDimensionCount)
-REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(SqrtGrad, y * 0.5f / x,
-                                           kNchwDimensionCount)
+REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(FloorDiv, dml::Floor(x / y), 8)
+REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(SigmoidGrad, y* x*(1 - x), 8)
+REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(TanhGrad, y*(1 - x * x), 8)
+REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(SqrtGrad, y * 0.5f / x, 8)
 REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(RsqrtGrad, y*(-0.5f * x) * (x * x),
-                                           kNchwDimensionCount)
-REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(ReciprocalGrad, -y* x* x,
-                                           kNchwDimensionCount)
-REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(InvGrad, -y* x* x,
-                                           kNchwDimensionCount)
+                                           8)
+REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(ReciprocalGrad, -y* x* x, 8)
+REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(InvGrad, -y* x* x, 8)
```

Comment:
```suggestion
REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(InvGrad, (-y * x * x), 8)
```

Nit parens to avoid the autoformatter's braindeadness.